habs_portal/static/js/views/TOCView.js

```diff
@@ -43,14 +49,15 @@ var TOCView = Backbone.View.extend({
   render: function() {
     var self = this;
     this.$el.html(this.template());
-    this.categoryCollection.each(function(category) {
-      var relevantCollection = new StationCollection();
-      var relevantStations = self.stationCollection.filter(function(item) {
-        return _.indexOf(item.get('categories'),category.get('id')) >= 0;
+    this.parentCollection.each(function(parent) {
+      // create an empty collection based on an empty child
+      var relevantCollection = new Backbone.Collection(self.childCollection.filter(false));
```

Comment:
@lukecampbell This seems to work fine.  Not sure if it's proper BB, but so be it.